client/ayon_core/pipeline/load/utils.py

```diff
@@ -639,6 +639,14 @@ def get_representation_path_with_anatomy(repre_entity, anatomy):
     try:
         context = repre_entity["context"]
         context["root"] = anatomy.roots
+
+        # Auto-fix 'udim' being list of integers
+        # - This is a legacy issue for old representation entities,
+        #   added 24/07/10
+        udim = context.get("udim")
+        if isinstance(udim, list):
+            context["udim"] = udim[0]
```

Comment:
Can we make one `_fix_representation_context_compatibility` function that is called in both these places that does this logic? That way, if we ever run into another issue for this, we can just expand that function? May be nicer than having it duplicated now?

Kind of [like this in avalon](https://github.com/getavalon/core/blob/master/avalon/pipeline.py#L1215)?